.github/workflows/clang_format_verify.yml

```diff
@@ -0,0 +1,16 @@
+name: Clang-Format Lint
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
This is not doing a recursive checkout, so that the linting can happen on the whole of `src`